ui/Makefile

```diff
@@ -52,9 +58,12 @@ $(TYPESCRIPT_TARGET): $(shell find $(TS_ROOT)) npm.installed
 	$(NODE_MODULES)/.bin/tsc -p $(TS_ROOT)
 	$(NODE_MODULES)/.bin/tslint -c $(TS_ROOT)/tslint.json $(addprefix -f , $(shell find $(TS_ROOT) -name *.ts -not -path *typings*))
 
+$(TEST_TARGET): $(shell find $(TS_ROOT)) npm.installed
```

Comment:
No, this pulls the production code directly into the test.js file. So this needs to be rebuilt any time any ts file changes in the project.